.github/workflows/ci-tests-ifort.yml

```diff
@@ -109,45 +64,34 @@ jobs:
         run: |
           .github/common/get-regression-files.sh
 
-      - name: Check out the correct regression test branch
+      - name: Check out regression test branch
         run: |
           .github/common/checkout-regression-branch.sh
 
-      - name: Update flopy MODFLOW 6 classes
+      - name: Update flopy
         run: |
           .github/common/update-flopy.sh
 
-      - name: Build MODFLOW 6 (Linux or macOS)
-        if: runner.os == 'Linux' || runner.os == 'macOS'
+      - name: Build modflow6
+        if: runner.os != 'Windows'
         run: |
-          source /opt/intel/oneapi/setvars.sh
           meson setup builddir -Ddebug=false --prefix=$(pwd) --libdir=bin
           meson install -C builddir
           cd autotest
           pytest -v --durations=0 get_exes.py
 
-      - name: Build MODFLOW 6 (Windows)
+      - name: Build modflow6 (Windows)
         if: runner.os == 'Windows'
-        shell: cmd /C CALL {0}
         run: |
-          call micromamba activate modflow6
-          call ".github/intel-scripts/build_windows.bat"
-          meson setup builddir -Ddebug=false --prefix=%CD% --libdir=bin
+          # activating micromamba environment causes default link.exe to be found instead of MSVC linker
```

Comment:
More detail on the reason for this [here](https://github.com/modflowpy/install-intelfortran-action/pull/5)